src/components/privateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import routes from '../../constants/routes';
+
+const PrivateRoute = ({ component: Component, ...props }) => {
+  const authenticated = useSelector(state => state.user.auth);
```

Comment:
```suggestion
  const authenticated = useSelector(({ user: { auth }}) => auth);
```

Just a simple aesthetic enhancement